unlock-js/src/__tests__/helpers/nockHelper.js

```diff
@@ -127,7 +127,9 @@ export class NockHelper {
     this.anyRequestSetUp = true // detect http calls made before any mocks setup
     const cb = (...args) => this.logNock(args)
     return this.nockScope
-      .post('/', { jsonrpc: '2.0', id: this._rpcRequestId, method, params })
+      .post('/', body => {
+        return body.jsonrpc === '2.0' && body.method === method
```

Comment:
Would you mind adding a comment around why this is here/what this scenario means?